### PR TITLE
zaproxy deps vol 1

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by developer-guy** [comment]
we should solve the `*.la` files issue at the package level side:

- https://github.com/wolfi-dev/os/issues/10829

File: libgsf.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  - name: libgsf-doc
+    pipeline:
+      - uses: split/manpages
+    description: libgsf manpages
+
+# we use gitlab for this package, so we need to disable the release monitor otherwise CI fails
+update:
+  enabled: false
+  manual: true
```
Comment (rawlingsj) [inline libgsf.yaml:89]: We should still be able to use auto updates even if the source is on gitlab.

**Review by rawlingsj — CHANGES_REQUESTED** [review]

File: utmps.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      install -D -m755 wtmpd.initd "${{targets.destdir}}"/etc/init.d/wtmpd
+      install -D -m755 btmpd.initd "${{targets.destdir}}"/etc/init.d/btmpd
+      install -D -m755 utmp-init.initd "${{targets.destdir}}"/etc/init.d/utmp-init
+      install -D -m644 wtmpd.logrotate "${{targets.destdir}}"/etc/logrotate.d/wtmpd
+      install -D -m644 btmpd.logrotate "${{targets.destdir}}"/etc/logrotate.d/btmpd
+      install -D -m755 setup-utmp "${{targets.destdir}}"/sbin/setup-utmp
+      install -D -m644 utmps.pc "${{targets.destdir}}"/usr/lib/pkgconfig/utmps.pc
+
+      mkdir -p "${{targets.destdir}}/usr/share/doc"
+      cp -a "$builddir/doc" "${{targets.destdir}}/usr/share/doc/${{package.name}}"
```
Comment (rawlingsj) [inline utmps.yaml:56]: As there's docs shall we add a docs subpackage below?  i.e.

```
  - name: utmps-docs
    pipeline:
      - uses: split/infodir
      - uses: split/manpages
```

File: startup-notification.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    dependencies:
+      runtime:
+        - startup-notification
+        - libsm-dev
+        - xcb-util-dev
+    description: startup-notification dev
+
+# we use gitlab for this package, so we need to disable the release monitor otherwise CI fails
+update:
+  enabled: false
```
Comment (rawlingsj) [inline startup-notification.yaml:69]: looks like we can use release monitoring https://release-monitoring.org/project/10089/

File: librsvg.yaml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+          mv "${{targets.destdir}}"/usr/bin/rsvg-convert "${{targets.subpkgdir}}"/usr/bin
+
+  - name: rsvg-convert-doc
+    pipeline:
+      - uses: split/manpages
+    description: librsvg manpages
+
+# we use gitlab for this package, so we need to disable the release monitor otherwise CI fails
+update:
+  enabled: false
```
Comment (rawlingsj) [inline librsvg.yaml:92]: We can still use release monitoring https://release-monitoring.org/project/5420/

File: libgsf.yaml (hunk 1 above)
Comment (rawlingsj) [reply libgsf.yaml:89]: you can still configure the update section https://release-monitoring.org/project/1980/

and also include a nolint at the top of the file and the other's in this PR, like this https://github.com/wolfi-dev/os/blob/041b90b0027949e4ab82037d21181bdcb27858fe/gitlab-runner.yaml#L4C1-L4C45

Comment (developer-guy) [reply libgsf.yaml:89]: great learning! noted it!

File: libgsf.yaml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+        - bzip2-dev
+    description: libgsf dev
+
+  - name: libgsf-doc
+    pipeline:
+      - uses: split/manpages
+    description: libgsf manpages
+
+update:
+  enabled: true
```
Comment (rawlingsj) [inline libgsf.yaml:88]:
```suggestion
  enabled: true
  version-transform:
    - match: _
      replace: .
```

Comment (developer-guy) [reply libgsf.yaml:88]: oh cool, didn't know that!

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**